include/rtl_433_devices.h

```diff
@@ -12,7 +12,6 @@
 		DECL(elv_em1000) \
 		DECL(elv_ws2000) \
 		DECL(lacrossetx) \
-		DECL(acurite5n1) \
```

Comment:
Hmm, I'm not sure what's better behavior: 
. silently changing protocol 9 to output the same thing as protocol 39
. "Breaking" protocol 9 to make it a silent NO-OP
. Throw an error if -R 9 is enabled (but not if -G is used.)

@mwall might have an opinion for how it affects his WeeWx driver.

Note: If this code gets enabled/run by using either -G or -R 9 -R 39, there is a subtle bug that rain will be counted twice.  Though see my next comment for changing the rain state keeping.